migrations/000011_delete_partition_function.up.sql

```diff
@@ -0,0 +1,18 @@
+CREATE OR REPLACE FUNCTION drop_ros_partition(tableDate TEXT)
+RETURNS void AS
+$BODY$
+DECLARE
+    allTables TEXT[];
```

Comment:
Will it be good to rename `allTables` to `partitionTables` and `tableName` to `partitionTable`?